pytermgui/ansi_interface.py

```diff
@@ -664,39 +750,39 @@ def report_mouse(
     """
 
     if event == "press":
-        _stdout.write("\x1b[?1000")
+        sys.stdout.write("\x1b[?1000")
 
     elif event == "highlight":
-        _stdout.write("\x1b[?1001")
+        sys.stdout.write("\x1b[?1001")
 
     elif event == "press_hold":
-        _stdout.write("\x1b[?1002")
+        sys.stdout.write("\x1b[?1002")
 
     elif event == "hover":
-        _stdout.write("\x1b[?1003")
+        sys.stdout.write("\x1b[?1003")
 
     else:
         raise NotImplementedError(f"Mouse report event {event} is not supported!")
 
-    _stdout.write("l" if stop else "h")
+    sys.stdout.write("l" if stop else "h")
 
     if method == "decimal_utf8":
-        _stdout.write("\x1b[?1005")
+        sys.stdout.write("\x1b[?1005")
 
     elif method == "decimal_xterm":
-        _stdout.write("\x1b[?1006")
+        sys.stdout.write("\x1b[?1006")
 
     elif method == "decimal_urxvt":
-        _stdout.write("\x1b[?1015")
+        sys.stdout.write("\x1b[?1015")
```

Comment:
I'm neutral to these changes. The reason `sys.stdout` is imported as `_stdout` is a remnant from before I had `__all__` set up, as I didn't want a star import to also pull in `sys`. In this current form this works fine as well.